### PR TITLE
Get back some vertical space in the style editor

### PR DESCRIPTION
**Pull request by aaime**
Removes the style name inside the page "contents" and puts it in the title instead

### CODE REVIEW

File: src/web/wms/src/main/resources/GeoServerApplication.properties [hunk 1]
```diff
@@ -46,7 +46,7 @@ KMLLayerConfigPanel.kmlFormatSettings           = KML Format Settings
 
 StyleEditPage.description = Edit the current style. The editor can provide syntax highlighting and automatic formatting. Click on the "validate" button to verify the style is a valid SLD document.
 StyleEditPage.name        = Name
-StyleEditPage.title       = Style Editor
+StyleEditPage.title       = Style Editor - {0}
```
Comment (tbarsballe) [inline src/web/wms/src/main/resources/GeoServerApplication.properties]: This seems to be causing an issue with the page/tab title - the wildcard replacement does not seem to be working there:
screen shot 2016-08-15 at 9 03 10 am

(Works fine in the UI though)

File: src/web/core/src/main/java/org/geoserver/web/css/geoserver.css [hunk 2]
```diff
@@ -1070,6 +1070,11 @@ height: 2.1em;
 margin: 1.5em 0 2.5em;
 }
 
+.tab-row-compact {
+margin: 1.5em 0 1em;
```
Comment (tbarsballe) [inline src/web/core/src/main/java/org/geoserver/web/css/geoserver.css]: Given the margin of the `page-header`, a style of `margin: -1em 0 1em;` might work better here (Or `margin: 0 0 1em;` at least) . Result:

screen shot 2016-08-15 at 9 18 25 am

**Comment by tbarsballe** [comment]
I quite like this change - I wonder if we could/should do something similar for the Layer/Layer Group edit pages, as they do the same thing the style page before this change.

File: src/web/wms/src/main/resources/GeoServerApplication.properties (hunk 1 above)
Comment (aaime) [inline src/web/wms/src/main/resources/GeoServerApplication.properties]: Ah, good catch, I did not notice. It seems there are two different template methods to override, getTitle and getPageTitle. I'll override both.

File: src/web/core/src/main/java/org/geoserver/web/css/geoserver.css (hunk 2 above)
Comment (aaime) [inline src/web/core/src/main/java/org/geoserver/web/css/geoserver.css]: I'll give it a try

**Comment by aaime** [comment]
I believe it sort of makes sense in general, except that the sections titles have a negative margin, so applying it to all pages might result some overlapping elements, like for example, in the "dimensions" tab of the layer editor.

**Comment by tbarsballe** [comment]
Ok, so something we could consider, but will likely need to be tested relatively thoroughly, and may involve rewriting some of the geoserver CSS. Something we can keep in mind for the future at least.

**Comment by aaime** [comment]
Page title fixed.

**Comment by tbarsballe** [comment]
Looks good to me.

**Pull request merged**